packages/chart-advisor/src/advisor/index.ts

```diff
@@ -1,14 +1,16 @@
-import { ChartKnowledgeJSON, CKBJson } from '@antv/ckb';
+import { CKBJson } from '@antv/ckb';
```

Comment:
`CKBJson` is a function, should its first letter be lowercase ?

Reply:
Yes, but this would be a Breaking Change, will do this later